molgenis-api-tests/src/test/java/org/molgenis/api/tests/rest/v2/RestControllerV2IT.java

```diff
@@ -279,7 +279,7 @@ public void testRetrieveSystemEntityCollectionAggregatesNotAllowed()
 			   .all()
 			   .header(X_MOLGENIS_TOKEN, testUserToken)
 			   .when()
-			   .get(API_V2 + "sys_App?aggs=x==active;y==superuser")
+			   .get(API_V2 + "sys_App?aggs=x==isActive")
```

Comment:
This was a funny one: because the user can read metadata, the system can figure out that the attribute is incorrect and returns a 404.